tests/subsys/fs/fs_api/src/main.c

```diff
@@ -72,6 +72,7 @@ void test_main(void)
 			 ztest_unit_test_setup_teardown(test_mount,
 							fs_setup,
 							dummy_teardown),
+			 ztest_unit_test(test_fs_file_t_init),
```

Comment:
Why isn't this in the commit that adds this test case?

Reply:
Some accidental addition to commit amend.